abce/trade.pyx

```diff
@@ -282,7 +282,7 @@ class Trade:
             offer.open_offer_status = 'polled'
             ret.append(offer)
         if sorted:
-            ret.sort(key=lambda objects: objects.price, reverse=descending, cmp=compare_with_ties)
+            ret.sort(key=lambda objects: objects.price, reverse=descending)
```

Comment:
There is limitation in python3's sort where only the key is available. There is `functools.cmp_to_key` (https://docs.python.org/3/library/functools.html#functools.cmp_to_key) but since the sorting function is implemented in cython, the `lambda x: x.price` can't be included.